server/odc-core/src/main/resources/i18n/BusinessMessages_zh_CN.properties

```diff
@@ -680,6 +680,21 @@ com.oceanbase.odc.TaskType.GENERATE_ROLLBACK=备份回滚方案生成
 com.oceanbase.odc.TaskType.PRE_CHECK=预检查
 com.oceanbase.odc.TaskType.EXPORT_RESULT_SET=导出结果集
 com.oceanbase.odc.TaskType.APPLY_PROJECT_PERMISSION=申请项目权限
+com.oceanbase.odc.TaskType.SQL_PLAN=SQL 计划
+com.oceanbase.odc.TaskType.DATA_ARCHIVE=数据归档
+com.oceanbase.odc.TaskType.DATA_DELETE=数据清理
+
+#
+# TaskStatus
+#
+com.oceanbase.odc.TaskStatus.EXECUTION_SUCCEEDED=执行成功
+com.oceanbase.odc.TaskStatus.EXECUTION_FAILED=执行失败
+com.oceanbase.odc.TaskStatus.EXECUTION_TIMEOUT=执行超时
```

Comment:
may: "com.oceanbase.odc.task.event.execution_timeout=xxx"?

Reply:
done, renamed to 'com.oceanbase.odc.event.TASK.EXECUTION_SUCCEEDED.name'